src/app/components/participant/participant-room/participant-room.component.html

```diff
@@ -4,19 +4,25 @@
   </mat-grid-tile>
   <mat-grid-tile [colspan]="15" [rowspan]="8">
 
-    <div fxLayout="column">
+    <div fxLayout="column" class="tasks-container">
 
-      <mat-card *ngIf="taskToEstimate" fxFill>
+      <mat-card *ngIf="taskToEstimate">
         <h3 *ngIf="!estimationResult">Estimation</h3>
         <h3 *ngIf="estimationResult">Estimation Finished</h3>
-        <div  fxLayout="row" fxLayoutAlign="space-around center"  class="estimation">
-          <app-task-item [task]="taskToEstimate" ></app-task-item>
+        <div fxLayout="row" fxLayoutAlign="space-around center" class="estimation">
+          <app-task-item [task]="taskToEstimate"></app-task-item>
           <app-estimation *ngIf="!estimationResult" (estimated)="estimateTask($event)"></app-estimation>
           <app-estimation-result *ngIf="estimationResult" [values]="estimationResult"></app-estimation-result>
         </div>
       </mat-card>
 
-      <h3>Tasks</h3>
+      <h3 *ngIf="tasks.length > 0" style="margin-left: 2em">Tasks</h3>
+      <div *ngIf="tasks.length == 0" class="slacking-container">
+        <h3 class="slacking-header">Your product owner is slacking</h3>
+        <img src="../../../../assets/sleep.png">
```

Comment:
I know that webpack is built-in in Angular, but is there an option to customise it's configuration? 

If so, consider creating resource roots for your project, this way it would be easier to import some shared resources - you could exchange this `src` attribute for something like `shared/pictures/sleep.png`

Reply:
But it's just a quick thought about scalability of the project, no strong opinion here ;)